Command/CrawlCommand.php

```diff
@@ -212,4 +212,20 @@ private function areHostsEqual($firstUrl, $secondUrl)
 
         return ($firstHost === $secondHost);
     }
+
+    private function isUrlWhitelisted($url, array $whitelist)
```

Comment:
I don't think this is the best place to have this function. Maybe you could make it a static function in the crawlJob..